hbase-server/src/test/java/org/apache/hadoop/hbase/rsgroup/TestRSGroupsAdmin1.java

```diff
@@ -621,4 +622,38 @@ public void testTableConstraint() throws Exception {
     TEST_UTIL.deleteTable(tableName);
     ADMIN.deleteNamespace(ns);
   }
+
+  @Test
+  public void testHBASE25301() throws IOException, InterruptedException {
```

Comment:
Please change the test method name.

Reply:
I have changed the method name and updated the PR.